### DISCUSSION TITLE
Christmas special high rounds & records

### DISCUSSION THREAD

**Discussion by toiletman74** [nzp-team/nzportable#270] [Records and Speedruns]
Got a 41 on christmas special. Lost the run due to getting trapped in a corner while training.
![image](https://user-images.githubusercontent.com/91436591/227137831-6fe69e2a-0a08-4101-a79a-ca4b2f28eb63.png)

**Comment by toiletman74** [comment]
![image](https://user-images.githubusercontent.com/91436591/230702201-ef01087e-60a5-4d74-bfc7-89cdc119fd46.png)
new record, downed twice to me being dumb on 40 & 41

**Comment by toiletman74** [comment]
Got a 51, but I didn't screenshot it. I have the demo showing the whole run here [DEMO](https://drive.google.com/file/d/1tFEGnhDYfbJNccVSmYh0zCmHYcuV-cN4/view?usp=sharing). Lost the run due to a bug with disconnecting my mouse. I was really really hoping to get a 52 on this run as that would be a new record for the highest round ever achieved on nzp ever. But not all was lost cause this run really really developed the strategy for the map. I tested out a new "fast strat" in this run which would be SUPER SUPER fast for getting through rounds 1-35 and it worked! I also developed a new strat for late game that is faster and safer than all of the strats I've tried before. All in all even though the run was lost at 51, the run really taught me a lot and I'm super hyped up for the runs to come.

**Comment by toiletman74** [comment]
![image](https://user-images.githubusercontent.com/91436591/231589152-b18f36ea-7470-476f-84a1-d72f935452b4.png)
turns out 68 is the real record for highest rounds ever in nzp but I still hit my goal of 52

**Reply by PrimiseRichtofen** [reply]
DO you know how long that would take
